modules/nw-egressnetworkpolicy-about.adoc

```diff
@@ -37,6 +37,29 @@ ifdef::ovn[]
 - A protocol that is one of the following protocols: TCP, UDP, and SCTP
 endif::ovn[]
 
+[IMPORTANT]
+====
+You must allow access to the {product-title} API servers or connections to the API endpoints are blocked. For example, you must define egress firewall rules similar to the following:
+
+[source,yaml]
+----
+apiVersion: k8s.ovn.org/v1
+kind: EgressFirewall
+metadata:
+  name: allow-from-api-endpoints
+spec:
+  egress:
+  - to:
+      cidrSelector: 74.6.0.0/16
+    type: Allow
```

Comment:
Please remove this rule 74.6.0.0/16,  it has nothing to do with this API servers.